### PR TITLE
COG-744-UsersList Updates when added new users / updated existing users

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
#### Which User story does this relate to (link)?
-COG-744-Manage Users list does not reflect changes immediately (> 18 minutes sometimes)

#### Brief feature description
- UserList Updates with new added users and updated details.

### CODE REVIEW

File: app/javascript/views/usersList/UsersList.test.js [hunk 1]
```diff
@@ -234,34 +239,61 @@ describe('UsersList', () => {
...
+          lastName="some_value"
+          officeNames={['north']}
+          inputData={{ officeNames: ['north'] }}
+          loggedInUserAccount={{ county_name: 'SomeCountyName' }}
           selectedOfficesList={['somevalue']}
         />
       );
     });
 
+    it('', () => {
```
Comment (Nasir2712) [inline app/javascript/views/usersList/UsersList.test.js]: can you update this?

File: app/javascript/views/usersList/UsersList.test.js [hunk 2]
```diff
@@ -234,34 +239,61 @@ describe('UsersList', () => {
...
+      comp.instance().componentDidMount();
+      expect(mockSetSearch).toHaveBeenCalledWith([
+        { field: 'last_name', value: 'some_value' },
+        { field: 'office_ids', value: ['north'] },
+      ]);
+    });
```
Comment (Nasir2712) [inline app/javascript/views/usersList/UsersList.test.js]: componentDIdMount also has one more action about setPage, do we need that in expect statement as well?

Comment (Sushmareddy-OSI) [reply app/javascript/views/usersList/UsersList.test.js]: Yes we need.

File: app/javascript/views/usersList/UsersList.test.js [hunk 3]
```diff
@@ -318,7 +351,8 @@ describe('UsersList', () => {
           lastName="some_value"
           officeNames={['north']}
           inputData={{ officeNames: ['north'] }}
-        />
+        />,
+        { disableLifecycleMethods: true }
```
Comment (RamuKammagani-osi) [inline app/javascript/views/usersList/UsersList.test.js]: i think you don't need to disable the lifecycle method here

**Pull request merged**